frame/response/event.go

```diff
@@ -71,3 +72,18 @@ func ParseSchemaChange(b *frame.Buffer) *SchemaChange {
 		return &SchemaChange{}
 	}
 }
+
+func ParseEvent(b *frame.Buffer) frame.Response {
+	s := b.ReadString()
+	switch s {
+	case "TOPOLOGY_CHANGE":
+		return ParseTopologyChange(b)
+	case "STATUS_CHANGE":
+		return ParseStatusChange(b)
+	case "SCHEMA_CHANGE":
+		return ParseSchemaChange(b)
+	default:
+		log.Fatalf("event type not supported: %s", s)
```

Comment:
This is not how we handle errors here, this would exit the APP for no good reason.

Reply:
I find lot's of this.

```
	if Debug {
		log.Printf("unknown consistency: %v", v)
	}
```

Reply:
For now Debug is constant set in frame/debug.go. Should we add another debug constant or make one for all packages?